cli/cmd/plugin/unmanaged-cluster/cmd/utils.go

```diff
@@ -57,6 +59,17 @@ func SetupRootCommand(rootCmd *cobra.Command) {
 	rootCmd.SetHelpTemplate(helpTemplate)
 }
 
+// LoggingVerbosity will get the configured logging level for this command.
+func LoggingVerbosity(cmd *cobra.Command) int {
+	level, err := cmd.Flags().GetInt32("verbose")
```

Comment:
@jpmcb would love to get your cobra expertise to verify this is how this parent flag should be handled. Thanks!